packages/edit-post/src/components/layout/index.native.js

```diff
@@ -105,6 +107,14 @@ class Layout extends Component {
 		return (
 			<SafeAreaView style={ styles.container } onLayout={ this.onRootViewLayout }>
 				{ mode === 'text' ? this.renderHTML() : this.renderVisual() }
+				<SafeAreaView>
+					<View style={ { height: 44 } } />
```

Comment:
Probably it would be better to have this value in `styles` file if we want to be consistent?